addons/prometheus/_namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
There is already a namespace in the prometheus directory, with your change there would be two.

Reply:
yeah, spotted that.  I think there might actually be a case for moving the namespaces out into their own addon?

Reply:
Doesn't make sense to me that they'd be in a separate addon. Rationale?